src/backend/internal/cli/serve/serve.go

```diff
@@ -51,6 +58,9 @@ var ServeCmd = &cobra.Command{
 		users := validated.Group("/users")
 		{
 			users.PUT("/v1/logout", handler.User.Logout)
+			users.POST("/v1/uploadProfile", handler.User.UploadProfilePicture)
```

Comment:
the endpoints should `kebab-case` -- separated by hyphen.
please refer to https://stackoverflow.com/questions/10302179/hyphen-underscore-or-camelcase-as-word-delimiter-in-uris for reasons.